packaging/src/docker/Dockerfile

```diff
@@ -14,14 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-ARG BUILD_ENV
+ARG BUILD_ENV=unarchive
```

Comment:
unarchive means temp here? Maybe the staging is more expressive

Reply:
Sorry for the delay in responding.
You're absolutely right that "unarchive" is confusing. Both "staging" and "local" would be much more expressive names:

1. "staging" clearly indicates this is for development/testing workflows
2. Since this stage specifically uses jars from local builds rather than remote repositories, "local" might be the most accurate and intuitive choice. It tells developers they need to have the binaries locally.